### DISCUSSION TITLE
Video stream cache

### DISCUSSION THREAD

**Discussion by wangxin3924** [bluenviron/mediamtx#771] [Questions]
推流的时候我可以缓存指定时间么，比如缓存5秒，这个应该是推流的时候设置还是流媒体服务器设置呢
Can I cache the specified time during streaming, such as 5 seconds? Should this be set during streaming or by the streaming media server

**Comment by aler9** [comment]
Hello, in my opinion you better tune the client to increase caching. I don't know if your aim is improving stability or simply delaying the video, but there's an option for both in VLC and ffmpeg.

**Reply by wangxin3924** [reply]
thank you  I want to  simply delaying the video

[a comment by an automated account is not shown]
